ansible/wait_for_vps.sh

```
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+VPS_IP="45.76.4.171"
+VPS_USER="root"
+MAX_ATTEMPTS=20
+WAIT_TIME=30
+
+echo "🔍 Waiting for VPS $VPS_IP to come back online..."
+echo "⏰ Will check every $WAIT_TIME seconds for up to $(($MAX_ATTEMPTS * $WAIT_TIME / 60)) minutes"
```

Shellcheck (0.10.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)